src/VscPreviewExtension/package.json

```diff
@@ -1,21 +1,25 @@
 {
   "name": "DfmPreview",
   "displayName": "DfmPreview",
-  "description": "Preview the DocFX Flavored Markdown in Visual Studio Code",
-  "version": "0.0.9",
+  "description": "Preview the DocFX Flavored Markdown and tokenTree in Visual Studio Code",
```

Comment:
did line 39 also need to update?

---
In reply to: [77320848](https://github.com/dotnet/docfx/pull/664#discussion_r77320848) [](ancestors = 77320848)